### ISSUE TITLE
Text to Speech not working

### ISSUE THREAD

**Issue by TheGreatestJannet** [flathub/info.mumble.Mumble#4]
When using the flatpak version of Mumble, text to speech doesn't work the option is greyed out. Would it be possible to resolve this? It might be because some dependencies are missing.

**Comment by JBBgameich** [comment]
Yes, Text to speech is currently not built. Although it's possible that I find enough time for this to fix it on my own, a pull request would be very welcome. I'm happy to review and help with it of course. This feature doesn't have the highest priority for me since I'm not using text to speech myself (which is not supposed to mean it can't be useful to others).

**Comment by JBBgameich** [comment]
Started to work on https://github.com/flathub/info.mumble.Mumble/pull/5, but I'd still be happy if someone could take over working on this feature since espeak{-ng,} still don't compile in the flatpak environment, and I'm not sure if speechd can even work without at least one of them.

**Comment by Hartmnt** [comment]
If I understand [this thread](https://bugzilla.mozilla.org/show_bug.cgi?id=1638150) correctly, a lot has changed with regards to speech-dispatcher inside of Flatpaks. It would seem like adding ``libspeechd2`` and exposing ``--filesystem=xdg-run/speech-dispatcher:ro`` could be enough already(?)
